src/intervals/construction.jl

```diff
@@ -0,0 +1,237 @@
+"""
+    IntervalArithmetic.default_bound()
```

Comment:
I am not sure the word `bound` is the most descriptive, it sounds like it has to do with the interval being bounded or unbounded, maybe precision?